release_notes.md

```diff
@@ -3,10 +3,6 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Python Worker Version to [4.27.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.27.0)
-- Update Java Worker Version to [2.14.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.14.0)
-- Update Python Worker Version to [4.26.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.26.0)
-- Updated `Microsoft.Azure.Functions.DotNetIsolatedNativeHost` version to 1.0.8 (#9936)
 	- [Improvements to NetHost logging & worker.config](https://github.com/Azure/azure-functions-dotnet-worker/pull/2315)
```

Comment:
Can we remove these too?

Reply:
Updated release notes. Please check and approve.